server/controllers/controllerEvent.js

```diff
@@ -0,0 +1,71 @@
+import db from '../models/jsondata';
+/**
+ *
+ *
+ * @class Event
+ */
+class Event {
+    /**
+   *
+   *
+   * @param {any} req
+   * @param {any} res
+   * @returns {json} adds an event
+   * @memberof Event
+   */
+  addEvent(req, res){
+      const{
+        userId, title, date, time, venue, details, centerId
+      } = req.body;
+      if (!userId){
```

Comment:
Rather this long if statements, use a middleware to validate your request body before passing it to the addEvent function

Reply:
ok, I'll do some research on that.

Reply:
Will fix that in a while